packaging/installer/kickstart-static64.sh

```diff
@@ -188,64 +188,68 @@ NETDATA_INSTALLER_OPTIONS=""
-		opts="${opts} --accept"
-		shift 1
-	elif [ "${1}" = "--dont-start-it" ]; then
-		NETDATA_INSTALLER_OPTIONS="${NETDATA_INSTALLER_OPTIONS:+${NETDATA_INSTALLER_OPTIONS} }${1}"
-		shift 1
-	elif [ "${1}" = "--no-updates" ]; then
-		NETDATA_UPDATES=""
-		shift 1
-	elif [ "${1}" = "--auto-update" ]; then
-		true # This is the default behaviour, so ignore it.
-		shift 1
-	elif [ "${1}" = "--stable-channel" ]; then
-		RELEASE_CHANNEL="stable"
-		NETDATA_INSTALLER_OPTIONS="${NETDATA_INSTALLER_OPTIONS:+${NETDATA_INSTALLER_OPTIONS} }${1}"
-		shift 1
-	elif [ "${1}" = "--disable-telemetry" ]; then
-		NETDATA_INSTALLER_OPTIONS="${NETDATA_INSTALLER_OPTIONS:+${NETDATA_INSTALLER_OPTIONS} }${1}"
-		shift 1
-	elif [ "${1}" = "--local-files" ]; then
-		NETDATA_UPDATES="" # Disable autoupdates if using pre-downloaded files.
-		shift 1
-		if [ -z "${1}" ]; then
-			fatal "Option --local-files requires extra information. The desired tarball full filename is needed"
-		fi
-
-		NETDATA_LOCAL_TARBALL_OVERRIDE="${1}"
-		shift 1
-		if [ -z "${1}" ]; then
-			fatal "Option --local-files requires a pair of the tarball source and the checksum file"
-		fi
-
-		NETDATA_LOCAL_TARBALL_OVERRIDE_CHECKSUM="${1}"
-		shift 1
-	else
-		echo >&2 "Unknown option '${1}' or invalid number of arguments. Please check the README for the available arguments of ${0} and try again"
-		exit 1
-	fi
+  if [ "${1}" = "--dont-wait" ] || [ "${1}" = "--non-interactive" ] || [ "${1}" = "--accept" ]; then
+    opts="${opts} --accept"
+    shift 1
+  elif [ "${1}" = "--dont-start-it" ]; then
+    NETDATA_INSTALLER_OPTIONS="${NETDATA_INSTALLER_OPTIONS:+${NETDATA_INSTALLER_OPTIONS} }${1}"
+    shift 1
+  elif [ "${1}" = "--no-updates" ]; then
+    NETDATA_UPDATES=""
+    shift 1
+  elif [ "${1}" = "--auto-update" ]; then
+    true # This is the default behaviour, so ignore it.
+    shift 1
+  elif [ "${1}" = "--stable-channel" ]; then
+    RELEASE_CHANNEL="stable"
+    NETDATA_INSTALLER_OPTIONS="${NETDATA_INSTALLER_OPTIONS:+${NETDATA_INSTALLER_OPTIONS} }${1}"
+    shift 1
+  elif [ "${1}" = "--disable-telemetry" ]; then
+    NETDATA_INSTALLER_OPTIONS="${NETDATA_INSTALLER_OPTIONS:+${NETDATA_INSTALLER_OPTIONS} }${1}"
+    shift 1
+  elif [ "${1}" = "--local-files" ]; then
+    NETDATA_UPDATES="" # Disable autoupdates if using pre-downloaded files.
+    shift 1
+    if [ -z "${1}" ]; then
+      fatal "Option --local-files requires extra information. The desired tarball full filename is needed"
+    fi
+
+    NETDATA_LOCAL_TARBALL_OVERRIDE="${1}"
+    shift 1
+    if [ -z "${1}" ]; then
+      fatal "Option --local-files requires a pair of the tarball source and the checksum file"
+    fi
+
+    NETDATA_LOCAL_TARBALL_OVERRIDE_CHECKSUM="${1}"
+    shift 1
+  else
+    echo >&2 "Unknown option '${1}' or invalid number of arguments. Please check the README for the available arguments of ${0} and try again"
+    exit 1
+  fi
 done
 
+if [ x"$DO_NOT_TRACK" = x"1" ]; then
```

Comment:
Same as above. I have seen this pattern used elsewhere, but I don't think I understand the reasoning behind it. Could you explain a bit ?

```suggestion
if [ "$DO_NOT_TRACK" = "1" ]; then
```